src/cloud_what/providers/azure.py

```diff
@@ -107,12 +107,16 @@ def is_likely_running_on_cloud(self):
         # We know that Azure uses only HyperV
         if 'virt.host_type' in self.hw_info:
             if 'hyperv' in self.hw_info['virt.host_type']:
-                probability += 0.3
+                probability += 0.2
 
         if 'dmi.chassis.asset_tag' in self.hw_info and \
                 self.hw_info['dmi.chassis.asset_tag'] == '7783-7084-3265-9085-8269-3286-77':
             probability += 0.3
```

Comment:
I would rather increase probability in this case, because it is really specific for Azure ATM.

Reply:
This chassis.asset_tag does only exist with dmi-decode data. If python-dmidecode isn't installed, this data doesn't exist. 

Additionally, the chassis.asset_tag is used in https://github.com/candlepin/subscription-manager/blob/0b7a210611de1dee494f3d29c8411dd3efa5d6b0/src/cloud_what/providers/azure.py#L89 as the single source to find out if sub-man is running on azure.

Reply:
The package `python-dmidecode` is required on x86_64 platform and I believe that Azure supports only this platform. Thus I believe that there is really low probability that python-dmidecode would not be installed.